### PR TITLE
fix #6718 broken 'conda install --rev'

### LINKED ISSUE #6230: Write downgrade integration tests
In an integration test:

1. Create an environment with the current conda under test, but include an earlier version of conda in that environment.  Probably something like

        conda create -n _test-env python=2.7 itsdangers conda=X.X

2. Subprocess to that conda, and make sure that conda can operate on that environment.

                CONDA_ROOT/envs/_test-env/bin/conda install -p CONDA_ROOT/envs/_test-env/bin/conda flask

### LINKED ISSUE #6290: conda install --revision N REMOVED CONDA
Hello, 

Today I discovered that conda can remove itself, which seems like it shouldn't be possible. Here's what happened:

Conda was updated from 4.3.29 to 4.3.30 when I installed a package (spyder-ide) that was supposed to let me get a bash terminal in my Spyder GUI. However, the package didn't work, so I decided to remove it. I did so by reverting back to the previous revision rather than uninstalling it and its dependencies. BAD MISTAKE! As you can see from my revision history below, conda install --revision 36 _removed_ conda entirely instead of downgrading it. Now I don't have conda. 

Is there a fix besides reinstalling Anaconda?

Thanks.

2017-11-12 15:26:47  (rev 36)
     conda  {4.3.29 -> 4.3.30}
    +coloredlogs-7.1
    +humanfriendly-4.4
    +spyder-terminal-0.2.2 (spyder-ide)

2017-11-12 15:29:16  (rev 37)
    -coloredlogs-7.1
    -conda-4.3.30
    -humanfriendly-4.4
    -spyder-terminal-0.2.2 (spyder-ide)

### LINKED ISSUE #6718: conda install --rev is broken in 4.3
per https://github.com/ContinuumIO/anaconda-issues/issues/7855#issuecomment-357036226

### PR DESCRIPTION
**Pull request by kalefranz**
fix #6718
resolve #6230 
fix #6290

### CODE REVIEW

File: conda/plan.py [hunk 1]
```diff
@@ -806,7 +806,7 @@ def revert_actions(prefix, revision=-1, index=None):
     if state == curr:
         return {}
 
-    dists = (Dist(s) for s in state)
+    dists = tuple(Dist(s) for s in state)
```
Comment (kalefranz) [inline conda/plan.py]: This line is the fix for #6718.

**Comment by kalefranz** [comment]
@jjhelmus Meant to talk to you about this.  But thanks for reviewing already.  For PRs of mine where I've added the "Ready for Review" tag, if they look good to you, go ahead an (1) merge the PR (use a standard merge commit please, i.e. don't squash or rebase; it helps with the CHANGELOG right now) and (2) delete the branch.

Thanks!

**Comment by jjhelmus** [comment]
@kalefranz Merged and branch deleted.

**Comment by kalefranz** [comment]
Thanks @jjhelmus!

**Comment by fabriciorsf** [comment]
This command doesn't work to me:

```
(myenv) username@mycomputer:~/git/myrepo$ conda install --rev 3

PackagesNotFoundError: The following packages are missing from the target environment:
  - defaults::libxslt==1.1.28=0
  - defaults::twisted==17.5.0=py36_0
```

Follow list revisions from my environment:
```
(myenv) username@mycomputer:~/git/myrepo$ conda list --revisions
2018-07-25 21:43:42  (rev 0)
    +asn1crypto-0.24.0 (intel)
    +backcall-0.1.0 (intel)
    +backports-1.0 (intel)
    +bleach-2.1.3 (intel)
    +boost-1.65.1 (intel)
    +bzip2-1.0.6 (intel)
    +caffe-1.1.0 (intel)
    +certifi-2018.1.18 (intel)
    +cffi-1.11.5 (intel)
    +chardet-3.0.4 (intel)
    +cryptography-2.2.2 (intel)
    +cycler-0.10.0 (intel)
    +cython-0.28.2 (intel)
    +daal-2018.0.3.20180405 (intel)
    +decorator-4.3.0 (intel)
    +entrypoints-0.2.3 (intel)
    +fontconfig-2.13.0 (intel)
    +freetype-2.9 (intel)
    +get_terminal_size-1.0.0 (intel)
    +gflags-2.2.1 (intel)
    +glog-0.3.5 (intel)
    +h5py-2.7.1 (intel)
    +hdf5-1.10.1 (intel)
    +html5lib-1.0.1 (intel)
    +icc_rt-2018.0.3 (intel)
    +icu-59.1 (intel)
    +idna-2.6 (intel)
    +impi_rt-2018.0.3 (intel)
    +intelpython-2018.0.3 (intel)
    +intelpython3_core-2018.0.3 (intel)
    +intelpython3_full-2018.0.3 (intel)
    +ipp-2018.0.3 (intel)
    +ipykernel-4.6.1 (intel)
    +ipyparallel-6.0.2 (intel)
    +ipython-6.3.1 (intel)
    +ipython_genutils-0.2.0 (intel)
    +ipywidgets-7.0.0 (intel)
    +jedi-0.12.0 (intel)
    +jinja2-2.9.6 (intel)
    +jpeg-9c (intel)
    +jsonschema-2.6.0 (intel)
    +jupyter-1.0.0 (intel)
    +jupyter_client-5.1.0 (intel)
    +jupyter_console-5.1.0 (intel)
    +jupyter_core-4.4.0 (intel)
    +kiwisolver-1.0.1 (intel)
    +leveldb-1.20 (intel)
    +libffi-3.2.1 (intel)
    +libpng-1.6.34 (intel)
    +libsodium-1.0.16 (intel)
    +libtiff-4.0.9 (intel)
    +libuuid-1.0.3 (intel)
    +libxml2-2.9.5 (intel)
    +llvmlite-0.23.0 (intel)
    +lmdb-0.9.22 (intel)
    +markupsafe-1.0 (intel)
    +matplotlib-2.2.2 (intel)
    +mistune-0.7.4 (intel)
    +mkl-2018.0.3 (intel)
    +mkl-dnn-0.13 (intel)
    +mkl_fft-1.0.2 (intel)
    +mkl_random-1.0.1 (intel)
    +mlsl-1.0 (intel)
    +mpi4py-3.0.0 (intel)
    +mpmath-1.0.0 (intel)
    +nbconvert-5.2.1 (intel)
    +nbformat-4.4.0 (intel)
    +networkx-2.0 (intel)
    +nose-1.3.7 (intel)
    +notebook-5.2.2 (intel)
    +numba-0.38.0 (intel)
    +numexpr-2.6.4 (intel)
    +numpy-1.14.3 (intel)
    +olefile-0.44 (intel)
    +opencv-3.1.0 (intel)
    +openmp-2018.0.3 (intel)
    +openssl-1.0.2o (intel)
    +pandas-0.22.0 (intel)
    +pandocfilters-1.4.1 (intel)
    +parso-0.2.0 (intel)
    +path.py-11.0.1 (intel)
    +pexpect-4.2.1 (intel)
    +pickleshare-0.7.4 (intel)
    +pillow-5.1.0 (intel)
    +pip-9.0.3 (intel)
    +pixman-0.34.0 (intel)
    +portpicker-1.1.1 (intel)
    +prompt_toolkit-1.0.15 (intel)
    +protobuf-3.5.2 (intel)
    +ptyprocess-0.5.2 (intel)
    +pycparser-2.18 (intel)
    +pydaal-2018.0.3.20180405 (intel)
    +pygments-2.2.0 (intel)
    +pyopenssl-17.5.0 (intel)
    +pyparsing-2.2.0 (intel)
    +pysocks-1.6.7 (intel)
    +pytables-3.4.2 (intel)
    +python-3.6.3 (intel)
    +python-dateutil-2.6.0 (intel)
    +python-gflags-3.1.1 (intel)
    +python-leveldb-0.194 (intel)
    +pytz-2018.4 (intel)
    +pywavelets-0.5.2 (intel)
    +pyyaml-3.12 (intel)
    +pyzmq-16.0.2 (intel)
    +requests-2.18.4 (intel)
    +scikit-image-0.13.1 (intel)
    +scikit-learn-0.19.1 (intel)
    +scipy-1.0.1 (intel)
    +setuptools-39.0.1 (intel)
    +simplegeneric-0.8.1 (intel)
    +six-1.11.0 (intel)
    +snappy-1.1.7 (intel)
    +sqlite-3.23.1 (intel)
    +sympy-1.1.1 (intel)
    +tbb-2018.0.4 (intel)
    +tbb4py-2018.0.4 (intel)
    +tcl-8.6.4 (intel)
    +tensorflow-1.3.1 (intel)
    +terminado-0.8.1 (intel)
    +testpath-0.3.1 (intel)
    +theano-1.1 (intel)
    +tk-8.6.4 (intel)
    +tornado-4.5.2 (intel)
    +traitlets-4.3.2 (intel)
    +urllib3-1.22 (intel)
    +wcwidth-0.1.7 (intel)
    +weakref-1.0.post1 (intel)
    +webencodings-0.5.1 (intel)
    +werkzeug-0.14.1 (intel)
    +wheel-0.31.0 (intel)
    +widgetsnbextension-3.2.0 (intel)
    +xgboost-0.7 (intel)
    +xz-5.2.3 (intel)
    +yaml-0.1.7 (intel)
    +zeromq-4.2.3 (intel)
    +zlib-1.2.11 (intel)

2018-07-25 22:49:35  (rev 1)
    +mkl-devel-2018.0.3 (intel)
    +mkl-include-2018.0.3 (intel)

2018-07-25 22:51:22  (rev 2)
    +boto-2.49.0
    +boto3-1.7.62
    +botocore-1.10.62
    +docutils-0.14
    +gensim-3.4.0
    +jmespath-0.9.3
    +libgcc-ng-7.2.0
    +mkl-service-1.1.2
    +nltk-3.3.0
    +patsy-0.5.0
    +s3transfer-0.1.13
    +seaborn-0.9.0
    +smart_open-1.6.0
    +statsmodels-0.9.0

2018-07-25 22:57:16  (rev 3)
    +atomicwrites-1.1.5
    +attrs-18.1.0
    +automat-0.7.0
    +beautifulsoup4-4.6.0
    +constantly-15.1.0
    +cssselect-1.0.3
    +hyperlink-18.0.0
    +incremental-17.5.0
    +libgpuarray-0.7.6
    +libxslt-1.1.28
    +lxml-4.1.1
    +mako-1.0.7
    +more-itertools-4.2.0
    +parsel-1.4.0
    +pluggy-0.6.0
    +py-1.5.4
    +pyasn1-0.4.3
    +pyasn1-modules-0.2.2
    +pydispatcher-2.0.5
    +pygpu-0.7.6
    +pytest-3.6.3
    +pytest-runner-4.2
    +queuelib-1.5.0
    +scrapy-1.5.0
    +service_identity-17.0.0
    +twisted-17.5.0
    +w3lib-1.19.0
    +zope-1.0
    +zope.interface-4.5.0

2020-08-03 17:48:28  (rev 4)
     asn1crypto  {0.24.0 (intel) -> 1.3.0 (defaults/linux-64)}
     atomicwrites  {1.1.5 -> 1.1.5 (defaults/linux-64)}
     attrs  {18.1.0 -> 19.3.0 (defaults/noarch)}
     automat  {0.7.0 -> 0.8.0 (defaults/noarch)}
     backcall  {0.1.0 (intel) -> 0.2.0 (defaults/noarch)}
     backports  {1.0 (intel) -> 1.0 (intel/linux-64)}
     beautifulsoup4  {4.6.0 -> 4.9.1 (defaults/linux-64)}
     bleach  {2.1.3 (intel) -> 3.1.5 (defaults/noarch)}
     boost  {1.65.1 (intel) -> 1.65.1 (intel/linux-64)}
     boto  {2.49.0 -> 2.49.0 (defaults/linux-64)}
     boto3  {1.7.62 -> 1.7.62 (defaults/linux-64)}
     botocore  {1.10.62 -> 1.10.62 (defaults/linux-64)}
     bzip2  {1.0.6 (intel) -> 1.0.8 (intel/linux-64)}
     caffe  {1.1.0 (intel) -> 1.1.0 (intel/linux-64)}
     certifi  {2018.1.18 (intel) -> 2020.4.5.2 (defaults/linux-64)}
     cffi  {1.11.5 (intel) -> 1.14.0 (intel/linux-64)}
     chardet  {3.0.4 (intel) -> 3.0.4 (intel/linux-64)}
     constantly  {15.1.0 -> 15.1.0 (defaults/linux-64)}
     cryptography  {2.2.2 (intel) -> 2.9.2 (intel/linux-64)}
     cssselect  {1.0.3 -> 1.1.0 (defaults/noarch)}
     cycler  {0.10.0 (intel) -> 0.10.0 (intel/linux-64)}
     cython  {0.28.2 (intel) -> 0.29.17 (intel/linux-64)}
     daal  {2018.0.3.20180405 (intel) -> 2020.2 (intel/linux-64)}
     decorator  {4.3.0 (intel) -> 4.4.2 (defaults/noarch)}
     docutils  {0.14 -> 0.14 (defaults/linux-64)}
     entrypoints  {0.2.3 (intel) -> 0.3 (defaults/linux-64)}
     fontconfig  {2.13.0 (intel) -> 2.13.0 (defaults/linux-64)}
     freetype  {2.9 (intel) -> 2.10.2 (intel/linux-64)}
     get_terminal_size  {1.0.0 (intel) -> 1.0.0 (intel/linux-64)}
     gflags  {2.2.1 (intel) -> 2.2.1 (intel/linux-64)}
     glog  {0.3.5 (intel) -> 0.3.5 (intel/linux-64)}
     h5py  {2.7.1 (intel) -> 2.10.0 (defaults/linux-64)}
     hdf5  {1.10.1 (intel) -> 1.10.6 (defaults/linux-64)}
     html5lib  {1.0.1 (intel) -> 1.0.1 (intel/linux-64)}
     hyperlink  {18.0.0 -> 19.0.0 (defaults/noarch)}
     icc_rt  {2018.0.3 (intel) -> 2020.2 (intel/linux-64)}
     icu  {59.1 (intel) -> 58.2 (defaults/linux-64)}
     idna  {2.6 (intel) -> 2.9 (intel/linux-64)}
     impi_rt  {2018.0.3 (intel) -> 2019.8 (intel/linux-64)}
     incremental  {17.5.0 -> 17.5.0 (defaults/linux-64)}
     intelpython  {2018.0.3 (intel) -> 2020.2 (intel/linux-64)}
     intelpython3_core  {2018.0.3 (intel) -> 2020.2 (intel/linux-64)}
     intelpython3_full  {2018.0.3 (intel) -> 2020.2 (intel/linux-64)}
     ipp  {2018.0.3 (intel) -> 2020.2 (intel/linux-64)}
     ipykernel  {4.6.1 (intel) -> 5.3.4 (defaults/linux-64)}
     ipyparallel  {6.0.2 (intel) -> 6.0.2 (intel/linux-64)}
     ipython  {6.3.1 (intel) -> 7.16.1 (defaults/linux-64)}
     ipython_genutils  {0.2.0 (intel) -> 0.2.0 (defaults/linux-64)}
     ipywidgets  {7.0.0 (intel) -> 7.5.1 (defaults/noarch)}
     jedi  {0.12.0 (intel) -> 0.17.1 (defaults/linux-64)}
     jinja2  {2.9.6 (intel) -> 2.11.2 (defaults/noarch)}
     jmespath  {0.9.3 -> 0.9.3 (defaults/linux-64)}
     jpeg  {9c (intel) -> 9b (defaults/linux-64)}
     jsonschema  {2.6.0 (intel) -> 3.2.0 (defaults/linux-64)}
     jupyter  {1.0.0 (intel) -> 1.0.0 (defaults/linux-64)}
     jupyter_client  {5.1.0 (intel) -> 6.1.6 (defaults/noarch)}
     jupyter_console  {5.1.0 (intel) -> 6.1.0 (defaults/noarch)}
     jupyter_core  {4.4.0 (intel) -> 4.6.3 (defaults/linux-64)}
     kiwisolver  {1.0.1 (intel) -> 1.2.0 (intel/linux-64)}
     leveldb  {1.20 (intel) -> 1.20 (intel/linux-64)}
     libffi  {3.2.1 (intel) -> 3.3 (intel/linux-64)}
     libgcc-ng  {7.2.0 -> 7.2.0 (defaults/linux-64)}
     libgpuarray  {0.7.6 -> 0.7.6 (defaults/linux-64)}
     libpng  {1.6.34 (intel) -> 1.6.37 (intel/linux-64)}
     libsodium  {1.0.16 (intel) -> 1.0.18 (defaults/linux-64)}
     libtiff  {4.0.9 (intel) -> 4.1.0 (defaults/linux-64)}
     libuuid  {1.0.3 (intel) -> 1.0.3 (defaults/linux-64)}
     libxml2  {2.9.5 (intel) -> 2.9.10 (intel/linux-64)}
     libxslt  {1.1.28 -> 1.1.34 (defaults/linux-64)}
     llvmlite  {0.23.0 (intel) -> 0.32.1 (intel/linux-64)}
     lmdb  {0.9.22 (intel) -> 0.9.22 (intel/linux-64)}
     lxml  {4.1.1 -> 4.5.2 (defaults/linux-64)}
     mako  {1.0.7 -> 1.1.3 (defaults/noarch)}
     markupsafe  {1.0 (intel) -> 1.1.1 (defaults/linux-64)}
     matplotlib  {2.2.2 (intel) -> 3.1.2 (intel/linux-64)}
     mistune  {0.7.4 (intel) -> 0.8.4 (defaults/linux-64)}
     mkl  {2018.0.3 (intel) -> 2020.2 (intel/linux-64)}
     mkl-devel  {2018.0.3 (intel) -> 2018.0.1 (defaults/linux-64)}
     mkl-dnn  {0.13 (intel) -> 0.13 (intel/linux-64)}
     mkl-include  {2018.0.3 (intel) -> 2020.1 (defaults/linux-64)}
     mkl-service  {1.1.2 -> 2.3.0 (intel/linux-64)}
     mkl_fft  {1.0.2 (intel) -> 1.1.0 (intel/linux-64)}
     mkl_random  {1.0.1 (intel) -> 1.1.1 (intel/linux-64)}
     mlsl  {1.0 (intel) -> 1.0 (intel/linux-64)}
     more-itertools  {4.2.0 -> 8.4.0 (defaults/noarch)}
     mpi4py  {3.0.0 (intel) -> 3.0.3 (intel/linux-64)}
     mpmath  {1.0.0 (intel) -> 1.1.0 (defaults/linux-64)}
     nbconvert  {5.2.1 (intel) -> 5.6.1 (defaults/linux-64)}
     nbformat  {4.4.0 (intel) -> 5.0.7 (defaults/noarch)}
     networkx  {2.0 (intel) -> 2.0 (intel/linux-64)}
     nltk  {3.3.0 -> 3.5 (defaults/noarch)}
     nose  {1.3.7 (intel) -> 1.3.7 (defaults/linux-64)}
     notebook  {5.2.2 (intel) -> 6.0.3 (defaults/linux-64)}
     numba  {0.38.0 (intel) -> 0.49.1 (intel/linux-64)}
     numexpr  {2.6.4 (intel) -> 2.7.0 (intel/linux-64)}
     numpy  {1.14.3 (intel) -> 1.18.5 (intel/linux-64)}
     olefile  {0.44 (intel) -> 0.46 (defaults/linux-64)}
     opencv  {3.1.0 (intel) -> 3.1.0 (intel/linux-64)}
     openmp  {2018.0.3 (intel) -> 2018.0.3 (intel/linux-64)}
     openssl  {1.0.2o (intel) -> 1.1.1g (intel/linux-64)}
     pandas  {0.22.0 (intel) -> 0.25.3 (intel/linux-64)}
     pandocfilters  {1.4.1 (intel) -> 1.4.2 (defaults/linux-64)}
     parsel  {1.4.0 -> 1.4.0 (defaults/linux-64)}
     parso  {0.2.0 (intel) -> 0.7.0 (defaults/noarch)}
     path.py  {11.0.1 (intel) -> 11.0.1 (intel/linux-64)}
     patsy  {0.5.0 -> 0.5.1 (defaults/linux-64)}
     pexpect  {4.2.1 (intel) -> 4.8.0 (defaults/linux-64)}
     pickleshare  {0.7.4 (intel) -> 0.7.5 (defaults/linux-64)}
     pillow  {5.1.0 (intel) -> 7.1.2 (defaults/linux-64)}
     pip  {9.0.3 (intel) -> 20.1 (intel/linux-64)}
     pixman  {0.34.0 (intel) -> 0.34.0 (intel/linux-64)}
     pluggy  {0.6.0 -> 0.13.1 (defaults/linux-64)}
     portpicker  {1.1.1 (intel) -> 1.1.1 (intel/linux-64)}
     prompt_toolkit  {1.0.15 (intel) -> 3.0.5 (defaults/noarch)}
     protobuf  {3.5.2 (intel) -> 3.12.3 (defaults/linux-64)}
     ptyprocess  {0.5.2 (intel) -> 0.6.0 (defaults/linux-64)}
     py  {1.5.4 -> 1.9.0 (defaults/noarch)}
     pyasn1  {0.4.3 -> 0.4.8 (defaults/noarch)}
     pyasn1-modules  {0.2.2 -> 0.2.7 (defaults/noarch)}
     pycparser  {2.18 (intel) -> 2.20 (intel/linux-64)}
     pydaal  {2018.0.3.20180405 (intel) -> 2018.0.3.20180405 (intel/linux-64)}
     pydispatcher  {2.0.5 -> 2.0.5 (defaults/linux-64)}
     pygments  {2.2.0 (intel) -> 2.6.1 (defaults/noarch)}
     pygpu  {0.7.6 -> 0.7.6 (defaults/linux-64)}
     pyopenssl  {17.5.0 (intel) -> 19.1.0 (intel/linux-64)}
     pyparsing  {2.2.0 (intel) -> 2.4.7 (intel/linux-64)}
     pysocks  {1.6.7 (intel) -> 1.7.0 (intel/linux-64)}
     pytables  {3.4.2 (intel) -> 3.4.2 (intel/linux-64)}
     pytest  {3.6.3 -> 5.4.3 (defaults/linux-64)}
     pytest-runner  {4.2 -> 5.2 (defaults/noarch)}
     python  {3.6.3 (intel) -> 3.7.7 (intel/linux-64)}
     python-dateutil  {2.6.0 (intel) -> 2.8.1 (intel/linux-64)}
     python-gflags  {3.1.1 (intel) -> 3.1.1 (intel/linux-64)}
     python-leveldb  {0.194 (intel) -> 0.194 (intel/linux-64)}
     pytz  {2018.4 (intel) -> 2020.1 (intel/linux-64)}
     pywavelets  {0.5.2 (intel) -> 0.5.2 (intel/linux-64)}
     pyyaml  {3.12 (intel) -> 5.3 (intel/linux-64)}
     pyzmq  {16.0.2 (intel) -> 19.0.1 (defaults/linux-64)}
     queuelib  {1.5.0 -> 1.5.0 (defaults/linux-64)}
     requests  {2.18.4 (intel) -> 2.23.0 (intel/linux-64)}
     s3transfer  {0.1.13 -> 0.1.13 (defaults/linux-64)}
     scikit-image  {0.13.1 (intel) -> 0.13.1 (intel/linux-64)}
     scikit-learn  {0.19.1 (intel) -> 0.23.1 (intel/linux-64)}
     scipy  {1.0.1 (intel) -> 1.4.1 (intel/linux-64)}
     scrapy  {1.5.0 -> 1.6.0 (defaults/linux-64)}
     seaborn  {0.9.0 -> 0.10.1 (defaults/noarch)}
     service_identity  {17.0.0 -> 18.1.0 (defaults/linux-64)}
     setuptools  {39.0.1 (intel) -> 47.3.0 (defaults/linux-64)}
     simplegeneric  {0.8.1 (intel) -> 0.8.1 (intel/linux-64)}
     six  {1.11.0 (intel) -> 1.15.0 (intel/linux-64)}
     smart_open  {1.6.0 -> 1.6.0 (defaults/linux-64)}
     snappy  {1.1.7 (intel) -> 1.1.7 (intel/linux-64)}
     sqlite  {3.23.1 (intel) -> 3.32.1 (intel/linux-64)}
     statsmodels  {0.9.0 -> 0.9.0 (defaults/linux-64)}
     sympy  {1.1.1 (intel) -> 1.6.1 (defaults/linux-64)}
     tbb  {2018.0.4 (intel) -> 2020.3 (intel/linux-64)}
     tbb4py  {2018.0.4 (intel) -> 2020.3 (intel/linux-64)}
     tcl  {8.6.4 (intel) -> 8.6.9 (intel/linux-64)}
     tensorflow  {1.3.1 (intel) -> 2.2.0 (defaults/linux-64)}
     terminado  {0.8.1 (intel) -> 0.8.3 (defaults/linux-64)}
     testpath  {0.3.1 (intel) -> 0.4.4 (defaults/noarch)}
     theano  {1.1 (intel) -> 1.0.4 (defaults/linux-64)}
     tk  {8.6.4 (intel) -> 8.6.9 (intel/linux-64)}
     tornado  {4.5.2 (intel) -> 6.0.4 (defaults/linux-64)}
     traitlets  {4.3.2 (intel) -> 4.3.3 (defaults/linux-64)}
     twisted  {17.5.0 -> 20.3.0 (defaults/linux-64)}
     urllib3  {1.22 (intel) -> 1.25.9 (intel/linux-64)}
     w3lib  {1.19.0 -> 1.21.0 (defaults/noarch)}
     wcwidth  {0.1.7 (intel) -> 0.2.5 (defaults/noarch)}
     weakref  {1.0.post1 (intel) -> 1.0.post1 (intel/linux-64)}
     webencodings  {0.5.1 (intel) -> 0.5.1 (defaults/linux-64)}
     werkzeug  {0.14.1 (intel) -> 1.0.1 (defaults/noarch)}
     wheel  {0.31.0 (intel) -> 0.34.2 (intel/linux-64)}
     widgetsnbextension  {3.2.0 (intel) -> 3.5.1 (defaults/linux-64)}
     xgboost  {0.7 (intel) -> 1.1.1 (intel/linux-64)}
     xz  {5.2.3 (intel) -> 5.2.5 (intel/linux-64)}
     yaml  {0.1.7 (intel) -> 0.1.7 (intel/linux-64)}
     zeromq  {4.2.3 (intel) -> 4.3.2 (defaults/linux-64)}
     zlib  {1.2.11 (intel) -> 1.2.11.1 (intel/linux-64)}
     zope  {1.0 -> 1.0 (defaults/linux-64)}
     zope.interface  {4.5.0 -> 4.5.0 (defaults/linux-64)}
    -gensim-3.4.0
    +_libgcc_mutex-0.1 (defaults/linux-64)
    +_tflow_select-2.3.0 (defaults/linux-64)
    +absl-py-0.9.0 (defaults/linux-64)
    +astunparse-1.6.3 (defaults/noarch)
    +bcrypt-3.1.7 (defaults/linux-64)
    +binutils_impl_linux-64-2.33.1 (defaults/linux-64)
    +binutils_linux-64-2.33.1 (defaults/linux-64)
    +blas-1.0 (defaults/linux-64)
    +blinker-1.4 (defaults/linux-64)
    +c-ares-1.15.0 (defaults/linux-64)
    +cachetools-4.1.0 (defaults/noarch)
    +click-7.1.2 (defaults/noarch)
    +daal4py-0.2020.2 (intel/linux-64)
    +dbus-1.13.16 (defaults/linux-64)
    +defusedxml-0.6.0 (defaults/noarch)
    +expat-2.2.9 (defaults/linux-64)
    +fastcache-1.1.0 (defaults/linux-64)
    +funcsigs-1.0.2 (intel/linux-64)
    +gast-0.3.3 (defaults/noarch)
    +gcc_impl_linux-64-7.3.0 (defaults/linux-64)
    +gcc_linux-64-7.3.0 (defaults/linux-64)
    +glib-2.65.0 (defaults/linux-64)
    +gmp-6.1.2 (defaults/linux-64)
    +gmpy2-2.0.8 (defaults/linux-64)
    +google-auth-1.17.2 (defaults/noarch)
    +google-auth-oauthlib-0.4.1 (defaults/noarch)
    +google-pasta-0.2.0 (defaults/noarch)
    +grpcio-1.27.2 (defaults/linux-64)
    +gst-plugins-base-1.14.0 (defaults/linux-64)
    +gstreamer-1.14.0 (defaults/linux-64)
    +gxx_impl_linux-64-7.3.0 (defaults/linux-64)
    +gxx_linux-64-7.3.0 (defaults/linux-64)
    +importlib-metadata-1.7.0 (defaults/linux-64)
    +importlib_metadata-1.7.0 (defaults/noarch)
    +intel-openmp-2020.2 (intel/linux-64)
    +joblib-0.15.1 (intel/linux-64)
    +keras-preprocessing-1.1.0 (defaults/noarch)
    +ld_impl_linux-64-2.33.1 (defaults/linux-64)
    +libgfortran-ng-7.3.0 (defaults/linux-64)
    +libprotobuf-3.12.3 (defaults/linux-64)
    +libstdcxx-ng-9.1.0 (defaults/linux-64)
    +libxcb-1.14 (defaults/linux-64)
    +lz4-c-1.9.2 (defaults/linux-64)
    +markdown-3.1.1 (defaults/linux-64)
    +mpc-1.1.0 (defaults/linux-64)
    +mpfr-4.0.2 (defaults/linux-64)
    +numpy-base-1.18.5 (intel/linux-64)
    +oauthlib-3.1.0 (defaults/noarch)
    +opt_einsum-3.1.0 (defaults/noarch)
    +packaging-20.4 (defaults/noarch)
    +pandoc-2.10 (defaults/linux-64)
    +pcre-8.44 (defaults/linux-64)
    +prometheus_client-0.8.0 (defaults/noarch)
    +prompt-toolkit-3.0.5 (defaults/noarch)
    +pyeditline-2.0.1 (intel/linux-64)
    +pyhamcrest-2.0.2 (defaults/noarch)
    +pyjwt-1.7.1 (defaults/linux-64)
    +pyqt-5.9.2 (defaults/linux-64)
    +pyrsistent-0.16.0 (defaults/linux-64)
    +qt-5.9.7 (defaults/linux-64)
    +qtconsole-4.7.5 (defaults/noarch)
    +qtpy-1.9.0 (defaults/noarch)
    +regex-2020.6.8 (defaults/linux-64)
    +requests-oauthlib-1.3.0 (defaults/noarch)
    +rsa-4.0 (defaults/noarch)
    +send2trash-1.5.0 (defaults/linux-64)
    +sip-4.19.8 (defaults/linux-64)
    +smp-0.1.4 (intel/linux-64)
    +soupsieve-2.0.1 (defaults/noarch)
    +tensorboard-2.2.1 (defaults/noarch)
    +tensorboard-plugin-wit-1.6.0 (defaults/noarch)
    +tensorflow-base-2.2.0 (defaults/linux-64)
    +tensorflow-estimator-2.2.0 (defaults/noarch)
    +termcolor-1.1.0 (defaults/linux-64)
    +tqdm-4.47.0 (defaults/noarch)
    +wrapt-1.12.1 (defaults/linux-64)
    +zipp-3.1.0 (defaults/noarch)
    +zstd-1.4.5 (defaults/linux-64)
```

**Pull request merged**